src/Lean/Server/Utils.lean

```diff
@@ -6,6 +6,7 @@ Authors: Wojciech Nawrocki, Marc Huisinga
 -/
 import Lean.Data.Position
 import Lean.Data.Lsp
+import Lean.Server.InfoUtils
```

Comment:
Maybe better to define `String.Range` (and `contains`) here and import `Utils` from `InfoUtils`?

Reply:
Oh, yes!

Reply:
Actually, should we replace it with `Substring`? I don't think we will ever need it on non-original trees.

Reply:
There are quite a few uses of `getRange?` in `RequestHandling`, are those guaranteed only to be on original trees?

Reply:
I believe so. The only source of non-original trees in the server is from `Info`s and `Info.range?` already uses `originalOnly`.